utils/mega_data_setup.py

```
@@ -171,7 +171,8 @@ def check_access_token(self):
 
     def __set_pet_names(self):
         pet_info = requests.get(
-            f"https://us.api.blizzard.com/data/wow/pet/index?namespace=static-us&locale=en_US&access_token={self.access_token}"
+            f"https://us.api.blizzard.com/data/wow/pet/index?namespace=static-us&locale=en_US",
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)